gitlab-pipeline/stage/build.yml

```diff
@@ -193,7 +193,7 @@ build:mender-artifact:
   services:
     - docker:dind
   tags:
-    - mender-qa-slave-highcpu
+    - mender-qa-worker-generic
```

Comment:
This could even be moved to a single core IMHO. I bet the difference won't be that great, and I think the net effect is negligible since most jobs have to wait for `build:servers` anyway.